src/Compilers/Test/Core/Traits/CompilerFeature.cs

```diff
@@ -43,5 +43,6 @@ public enum CompilerFeature
         RecordStructs,
         RequiredMembers,
         RefLifetime,
+        Extensions,
     }
 }
```

Comment:
incremental tests that changing `class C(int i)` to `extension(int i)` and vvice versa works would be good.